src/hlstrafe.cpp

```diff
@@ -2117,6 +2130,53 @@ namespace HLStrafe
 
 			curState.Parameters.Parameters.Yaw.Yaw = newValue;
 		}
+
+		// This would only matters when we have some acceleration value for this type of strafing.
+		// Otherwise the final value would not change.
+		curState.MaxAccelYawOffset = frame.GetType() == HLTAS::StrafeType::MAXACCELYAWOFFSET;
+		if (curState.MaxAccelYawOffset) {
+			const auto frame_start = static_cast<float>(frame.GetMaxAccelYawOffsetStart());
+			const auto frame_target = static_cast<float>(frame.GetMaxAccelYawOffsetTarget());
+			const auto frame_accel =  static_cast<float>(frame.GetMaxAccelYawOffsetAccel());
+			const auto frame_dir = frame.GetDir();
+
+			const auto frame_reset = frame_start != curState.MaxAccelYawOffsetStart
+				|| frame_target != curState.MaxAccelYawOffsetTarget
+				|| frame_accel != curState.MaxAccelYawOffsetAccel
+				|| frame_dir != curState.MaxAccelYawOffsetDir
+				;
+
+			// Resets if we have a frame with different values.
+			// Stays the same if we don't though.
+			if (frame_reset) {
+				// Slightly offset them as a hack to get 0 and frame_target value.
```

Comment:
Maybe just put the calculation below in an `} else {` of this if? This way there's no need to offset here